modules/ROOT/pages/api-analytics-dashboard.adoc

```diff
@@ -91,6 +94,10 @@ Request Volume:: The sum of requests in the selected time period for the given A
 
 Failed Requests:: The sum of failed requests in the selected time period for the given API. Data is aggregated in one minute increments.
 
+Total 4xx Errors:: The sum of 400 errors (such as 404, 408, and so on) in the selected time period for the given API.
+
+Total 5xx Errors:: The sum of 500 errors (such as 500, 502, and so on) in the selected time period for the given API.
```

Comment:
Does this mean the sum of 404+408+... or do you mean the total number of errors in the 400-499 range? 
Ditto for 5xx.

Reply:
Total errors that are labeled as 400- or. 500-level errors. So things like 404 (page not found error), 500 (HTTP request denied), and so on. Networking speak.